General edits to resampling method (#18)
resampling-stats/resampling-with: source/resampling_method.Rmd

so that it will be there when the computer receives the next command to `print`
that result on the screen.

Don't worry too much about the details of how each of these commands works --- we
will cover those details gradually, over the next few chapters.  But, we hope
that you can see, in principle, how each of the operations that the computer
carries out are analogous to the operations that you yourself executed when you

> solved this problem using a ten-sided die. This is

Reviewer: ```suggestion
solved this problem using the equivalent of a ten-sided die. This is
```

the correct formula---considerable hard thinking, and perhaps some
digging in textbooks. More important than the labor, however, is that
you may come up with the wrong formula, and hence obtain the wrong
answer. And how would you know if you were wrong?

Most students who have had a standard course in probability and
statistics are quick to tell you that it is not easy to find the correct
formula, even immediately after finishing a course (or several courses)

> on the subject. After leaving school, it is harder still to choose the

Reviewer: ```suggestion
on the subject. After leaving school or university it is harder still to choose the
```

right formula. Even many people who have taught statistics at the
university level (including this writer) must look at a book to get the
correct formula for a problem as simple as the ambulances, and then we are

> still often unsure of the right answer. This is the grave disadvantage of

Reviewer: ```suggestion
often still not sure we have the right answer. This is the grave disadvantage of
```
